cypress/support/step-definitions/common-steps.js

```diff
@@ -125,10 +126,23 @@ When('I hover mouse onto icon', () => {
   icon().trigger('mouseover');
 });
 
+When('I hover mouse onto icon into iFrame', () => {
+  cy.wait(100, { log: DEBUG_FLAG }); // required becasue element might be reloaded
+  iconNoIframe().trigger('mouseover');
+});
+
+Then('I hover mouse onto {string} icon', (name) => {
+  getElementNoIframe(name).trigger('mouseover');
+});
```

Comment:
probably we could use one of this functions above

Reply:
fixed